client/src/js/services/ModalService.js

```diff
@@ -25,9 +25,14 @@ ModalService.$inject = [ '$uibModal' ];
  */
 function ModalService(Modal) {
   var service = this;
-  
+
   service.alert = alert;
-  service.confirm = confirm;  
+  service.confirm = confirm;
```

Comment:
This is a very nice iteration.  Having all modals in one place is a nice optimization and makes it much easier to develop.